src/_data/catalog/widgets.json

```diff
@@ -1615,6 +1868,17 @@
     "link": "https://api.flutter.dev/flutter/material/SnackBar-class.html",
     "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-snack-bar.png'>"
   },
+  {
+    "name": "Snackbar",
+    "description": "Snackbars provide brief messages about app processes at the bottom of the screen.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Brief messages about app processes that display at the bottom of the screen.",
```